enclave_manager/avalon_enclave_manager/enclave_manager.py

```diff
@@ -168,104 +190,106 @@ def process_work_orders(self, kv_helper):
         try:
             # Get all workorders requests from KV storage lookup and process
             list_of_workorders = kv_helper.lookup("wo-scheduled")
-
             if not list_of_workorders:
                 logger.info("Received empty list of work orders from " +
                             "wo-scheduled table")
                 return
-
         except Exception as e:
             logger.error("Problem while getting keys from wo-scheduled table")
             return
 
         for wo_id in list_of_workorders:
-            try:
-                kv_helper.set("wo-processing", wo_id,
-                              WorkOrderStatus.PROCESSING.name)
-
-                # Get JSON workorder request corresponding to wo_id
-                wo_json_req = kv_helper.get("wo-requests", wo_id)
-                if wo_json_req is None:
-                    logger.error("Received empty work order corresponding " +
-                                 "to id %s from wo-requests table", wo_id)
-                    kv_helper.remove("wo-processing", wo_id)
-                    return
-
-            except Exception as e:
-                logger.error("Problem while reading the work order %s"
-                             "from wo-requests table", wo_id)
-                kv_helper.remove("wo-processing", wo_id)
-                return
+            self.process_work_order_by_id(kv_helper, wo_id)
+        # end of for loop
+    # -----------------------------------------------------------------
 
```

Comment:
Keep 2 blanks lines between functions

Reply:
Couldn't add 2 lines between methods in a class.   Errors in lint check.

Reply:
Interesting, lint suggests to add 2 line breaks in between functions. I guess this is not applicable for class member functions.

Reply:
@manju956 IIRC, pylint wants 2 line breaks between functions, but 1 line breaks between methods within a class

Reply:
@danintel Thanks for the info.